examples/tts/conf/talknet-durs.yaml

```diff
@@ -148,6 +150,8 @@ model:
     sched:
       name: CosineAnnealing
       min_lr: 1e-5
+      warmup_ratio: 0.02
+      max_steps: ???
```

Comment:
I don't think you need this. This should be computed by NeMo itself.

Reply:
You mean I can remove `max_steps`?